style.css

```diff
@@ -0,0 +1,169 @@
+.main {
+  width: 375px;
+  height: 6968px;
+  background: #fff;
+}
+
+.header {
+  width: 375px;
+  height: 92px;
+  background: #3c3a39;
+  flex-direction: row;
+  order: 0;
+  left: 0;
+  top: 0;
+}
+
+.toolbar {
+  display: flex;
+  flex-direction: row;
+  position: absolute;
+  width: 375px;
+  height: 48px;
+  top: 44px;
+}
+
+.placeholder {
+  display: flex;
+  align-items: center;
+  position: absolute;
+  width: 101px;
+  height: 25px;
+  left: 24px;
+  top: calc(50% - 44px / 2);
+  border-radius: 8px;
+}
+
+#text {
+  width: 100px;
+  height: 24px;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.menu {
+  display: flex;
+  align-items: center;
+  padding: 2px;
+  position: absolute;
+  width: 25px;
+  height: 25px;
+  left: 300px;
+  border-radius: 8px;
+}
+
+#menu-icon {
+  position: absolute;
+  width: 18px;
+  height: 12px;
+  right: 6px;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  width: 375px;
+  height: 590px;
+  background: url('../Images/a.png') no-repeat #1c1a19;
+  background-position: 100% 100%;
+  order: 1;
+}
+
+.headline-fmt {
+  padding-left: 30px;
+}
+
+.primary-text {
+  width: 327px;
+  height: 124px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 56px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.supporting-text {
+  width: 327px;
+  height: 27px;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+  display: flex;
+  text-align: center;
+  color: #fff5e1;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.description {
+  text-align: left;
+  width: 327px;
+  height: 123px;
+  font-family: 'Poppins', 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+.social-media {
+  display: flex;
+  flex-direction: row;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+  list-style-type: none;
+  align-items: center;
+  justify-content: space-between;
+  width: 200px;
+  height: 40px;
+}
+
+a:hover {
+  background-color: #ff6b00;
+}
+
+#github-icon {
+  order: 0;
+  border-radius: 8px;
+}
+
+#linkedin-icon {
+  order: 1;
+  border-radius: 8px;
+}
+
+#angelist-icon {
+  order: 2;
+  border-radius: 8px;
+}
+
+#twitter-icon {
+  order: 3;
+  border-radius: 8px;
+}
+
+#medium-icon {
+  order: 4;
+  border-radius: 8px;
+}
```

Comment:
- [ ] Please kindly make sure your design stick as much as possible to the Figma design. You have some spaces to the left and to the top of your page, kindly fix it 🙃 🙃 

Your design

<img width="283" alt="Screen Shot 2022-10-28 at 22 31 40" src="https://user-images.githubusercontent.com/74501523/198728787-e200292f-387d-4800-9f31-9cfa4b50b36e.png">

Figma design

<img width="305" alt="Screen Shot 2022-10-28 at 22 42 50" src="https://user-images.githubusercontent.com/74501523/198728872-736e96b8-15a3-4400-a597-02980076fe7e.png">